ansible/docker/Jenkinsfile.test

```diff
@@ -0,0 +1,134 @@
+                    } 
+                    steps {
+                        dockerBuild('arm64', 'ubuntu2404', 'Dockerfile.u2404')
+                    }
+                }
+                stage('UBI10 x64') {
+                    agent {
+                        label "dockerBuild&&linux&&x64&&dockerhost-azure-ubuntu2204-x64-1"
+                    } 
+                    steps {
+                        dockerBuild('amd64', 'ubi10', 'Dockerfile.ubi10')
+                    }
+                }
+               stage('UBI10 aarch64') {
+                   agent {
+                       label "dockerBuild&&linux&&aarch64"
+                   }
+                   steps {
+                       dockerBuild('arm64', 'ubi10', 'Dockerfile.ubi10')
+                   }
+               }
+//               stage('UBI10 ppc64le') {
+//                   agent {
+//                       label "dockerBuild&&linux&&ppc64le"
+//                   }
+//                   steps {
+//                       dockerBuild('ppc64le', 'ubi10', 'Dockerfile.ubi10')
+//                   }
+//               }
+            }
+        }
+        stage('Docker Manifest') {
+            agent {
+                label "dockerBuild&&linux&&x64"
+            } 
+            environment {
+                DOCKER_CLI_EXPERIMENTAL = "enabled"
+            }
+            steps {
+                dockerManifest()
+            }
+        }
+        stage('cosign') {
+            agent {
+                label "cosign"
+            }
+            environment {
+                DOCKER_CLI_EXPERIMENTAL = "enabled"
+            }
+            steps {
+                  sh "rm -vf *.sha256"
+                  copyArtifacts( projectName: '${JOB_NAME}', selector: specific("${BUILD_ID}"), flatten: true )
+                  sh "ls -ld *.sha256"
+                  cosign()
+            }
+        }
+
+    } 
+} 
+
+def dockerBuild(architecture, distro, staticdockerfile) {
+    sh "rm -vf *.sha256"
+    git poll: false, url: 'https://github.com/adoptium/infrastructure.git'
+    def git_sha = "${env.GIT_COMMIT.trim()}"
+    dockerImage =
+    docker.build("ghcr.io/adoptium/test-containers:${distro}-${architecture}",
+        "-f ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/DockerStatic/Dockerfiles/$staticdockerfile .")
+    // dockerhub is the ID of the credentials stored in Jenkins 
+    docker.withRegistry('https://ghcr.io', 'ghcr-adoptium') {
+        dockerImage.push()
+        sh "docker inspect --format='{{index .RepoDigests 0}}' ghcr.io/adoptium/test-containers:${distro}-${architecture} > ${distro}_linux-${architecture}.sha256"
+        archiveArtifacts artifacts: '*linux*.sha256', fingerprint: true
+    }
+}
+
+def dockerManifest() { 
+    // dockerhub is the ID of the credentials stored in Jenkins
+    docker.withRegistry('https://ghcr.io', 'ghcr-adoptium') {
+        git poll: false, url: 'https://github.com/sxa/infrastructure.git'
```

Comment:
Are you still wanting to reference sxa/infrastructure here?

Reply:
To be honest I'm pretty sure those aren't needed so I'll run a test with those two lines commented out.